files_to_review/customQueryGenerator.js

```diff
@@ -100,13 +108,8 @@ function buildCustomQueryFunctionScript(propertySelector, selector){
 					}\
 					for (i=0; i<object.length; i++){\
 						var element = object[i].key["0"];\
```

Comment:
In this case, yes. The call to MixPanel's server is via a string variable. In order to show this string variable in the form of the intended function, instead of a super long string, the '\' are necessary to divide the string yet incorporate the elements to form one bigger string.